### DISCUSSION TITLE
How to set class attribute on <html> tag?

### DISCUSSION THREAD

**Discussion by wtho** [leptos-rs/leptos#547] [Q&A]
When using `<Html lang="en" class="..."/>`, the compiler complains:
```
error[E0599]: no method named `class` found for struct `HtmlPropsBuilder` in the current scope
  --> src/app.rs:14:25
   |
14 |         <Html lang="de" class="bg-primary-bg font-var h-full"/>
   |                         ^^^^^ method not found in `HtmlPropsBuilder<((Option<TextProp>,), ())>`
```
Is there a way to set the class attribute? Setting a class on `<html>` is valid in html 5.

**Comment by gbj** [comment]
Hmmm can't say I've ever seen this before to me honest. Two options
1. Set it on `<body>` with the `<Body/>` component.
2. Make a PR to `leptos_meta` to add the `class` prop to `<Html/>` on the same model as `<Body/>` -- I'd be happy to merge it!
